docs/_source/tutorials/notebooks/labelling_textclassification_setfit_zeroshot.ipynb

```diff
@@ -0,0 +1,645 @@
+        "\n",
+        "dataset = dataset.map(lambda batch: {\"prediction\": list(get_predictions(batch[\"text\"]))}, batched=True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-ZQMFaRLXywT"
+      },
+      "source": [
+        "\n",
+        "Let's upload our dataset with vectors and predictions."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Kv2qsk_1XywT"
+      },
+      "outputs": [],
+      "source": [
+        "rg_ds = rg.DatasetForTextClassification.from_datasets(dataset)\n",
+        "\n",
+        "rg.log(\n",
+        "    name=\"banking77-topics-setfit\",\n",
+        "    records=rg_ds,\n",
+        "    chunk_size=50,\n",
+        ")"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Y-e_l9dtXywV"
+      },
+      "source": [
+        "## 🏷 Bulk labelling with the `find similar` and zero-shot predictions\n",
+        "Now that our `banking77-topics-setfit` is available from the Argilla UI. You can start annotating the data leveraging similarity search and zero-shot predictions. After going to your Argilla UI URL, the workflow is following:\n",
+        "\n",
+        "1. Label a record (e.g., label \"Change my information\" as `change details`) and then click on Find similar on the top-right of your record.\n",
+        "2. As a result, you'll get to a list of the most similar record sorted by similarity with their corresponding predictions.\n",
+        "3. You can now review the predictions, validate them, or correct them.\n",
+        "\n",
+        "\n",
+        "After labelling around 200 records, we're ready to evaluate our zero-shot model, let's see how! "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "668C81adsaLK"
+      },
+      "source": [
+        "## 📏 Evaluate the zero-shot model\n",
+        "\n",
+        "We can use Argilla's built-in metrics to compute `f1` based on (1) the predictions of the zero-shot model we stored at the beginning of the tutorial, and (2) the manual annotations. Please note that during the labelling process, we've added a new label `Other` to account for examples that didn't fall into our predefined categories. This highlights the importance of iterating early on during project definition. Argilla gives users a lot of flexibility, features like predictions and similarity search can help to surface potential issues and refinements much faster than with traditional data annotation tools. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 542
+        },
+        "id": "6JFCGtGlsh0x",
+        "outputId": "911e0115-97a1-4a30-ec3b-1fa963a4f26c"
+      },
+      "outputs": [
```

Comment:
This cell its output does not want to show: https://docs.argilla.io/en/docs-adds_zeroshot_setfit_tutorial/tutorials/notebooks/labelling_textclassification_setfit_zeroshot.html

Reply:
I think it was caused by exporting from Colab. I tried now to rebuild the output locally now and we'll see if it renders